articles/what-is-an-ssl-certificate.mdx

```diff
@@ -0,0 +1,110 @@
+4. The browser contacts the web server again with a random string encrypted with the Public key from the SSL certificate.
+5. The web server uses the corresponding Private key to decrypt the message.
+6. If the web server and the user's browser agree on the strings for Client Random, Server Random, and the decrypted message, then the web server will allow the browser to access the website in an encrypted session.
+
+This process only takes milliseconds to complete, so it will not affect the user's experience on your website.
+
+SSL certificates last for about a year and should be renewed at yearly intervals to maintain your website's security. You can renew you SSL certificate more often, but most website do not require this level of security.
+
+Depending on your browser, a padlock icon will appear near your URL once you add an SSL certificate to your website. You can click on this icon to review the publicly available certificate information.
+
+## TLS Version
+
+To configure an SSL certificate for your Site on CloudCannon, you must specify the minimum TLS version.
+
+* **Minimum TLS Version** — The minimum version of TLS that a user's browser must have to be allowed to access your Site.
+
+A higher TLS version is a stronger cryptographic standard, including fixes for known vulnerabilities in previous versions. On CloudCannon, the minimum TLS version you can use for your Site is 1.2, however 1.3 is available if you require it.
+
+## Validation Method
+
+To configure an SSL certificate for your Site on CloudCannon, you must specify the validation method.
+
+* **Validation Method** — The method used by the Certificate Authority to authenticate you are the owner of your domain. This is sometimes called the DCV (Domain Control Validation) method.
+
+An important role of Certificate Authority is to ensure that the person requesting an SSL certificate is authorized to use the domain they request it for. On CloudCannon, you can verify ownership of a domain using one of the following methods: *DNS TXT Record* or *HTTP/Automatic*.
+
+DNS TXT Records demonstrate control over a domain, as only authorized individuals should be able to update the records stored by your DNS provider. If you select DNS TXT record as your validation method, CloudCannon will provide you with two TXT records to add to your DNS provider.
+
+HTTP/Automatic adds a verification HTTP token to your domain. If you select HTTP/Automatic as your validation method, you cannot use a [wildcard SSL certificate](/documentation/articles/https-support-for-your-sites-hosted-on-cloudcannon/).
+
+## SSL certificate types
+
+On CloudCannon, any Site with a [Custom Domain](/documentation/articles/what-is-a-custom-domain/) can use a [wildcard SSL certificate](/documentation/articles/https-support-for-your-sites-hosted-on-cloudcannon/) or a [custom SSL certificate](/documentation/articles/add-a-custom-ssl-certificate/).
```

Comment:
Site can also use single-domain SSL, which is the default for most to the sites.

I think it would be good to have a section talking about how/when CloudCannon auto-generates SSL for the site vs when they need to manually act on it.